apps/console/src/features/applications/components/edit-application.tsx

```diff
@@ -374,6 +375,18 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
         isInboundProtocolConfigRequestLoading
     ]);
 
+    useEffect(() => {
+
+        if (application?.advancedConfigurations?.additionalSpProperties?.length > 0) {
+            application?.advancedConfigurations?.additionalSpProperties?.
+            forEach((item, index) => {
+                if (item.name === "isEnterpriseLoginManagementApp" && item.value === "true") {
+                    setIsEnterpriseLoginMgt(true);
+                }
+            });
+        }
+    }, [application]);
```

Comment:
There should be a lint warning here. Check if ESLint is installed in the IDE.
```suggestion
    }, [ application ]);
```